apps/desktop/desktop/src/main/java/bisq/desktop/webcam/WebcamProcessLauncher.java

```diff
@@ -45,7 +45,12 @@ public WebcamProcessLauncher(String baseDir) {
     public CompletableFuture<Process> start(int port) {
         return CompletableFuture.supplyAsync(() -> {
             try {
-                String version = FileUtils.readStringFromResource("webcam-app/version.txt");
+                String version = "1.0.0";
+                try {
+                    version = FileUtils.readStringFromResource("webcam-app/version.txt");
```

Comment:
The version file is expected to exist (in dev mode it requires a gradle task to get called but in binary it must exist). so failing the whole code block seems correct to me instead of only logging an error.

Reply:
I was not supposed to push that change, it's always throwing exceptions when I use the camera locally...
I'll revert this in the next amend

Reply:
You need to run the task described in the readme in the webcam module if u want to try it out